dcc-import-gene/src/main/java/org/icgc/dcc/imports/gene/reader/EntrezXMLReader.java

```diff
@@ -92,7 +92,7 @@ private static SAXParserFactory createParserFactory()
 
     @Override
     public void startElement(String uri, String localName, String qName, Attributes attributes) {
-      if ("Entrezgene_summary".equals(qName)) {
+      if ("Entrezgene".equals(qName)) {
```

Comment:
Switching to the `Entrezgene` root, ensures that the logic happens within a given `Entrezgene` tag which eliminates the possibility of carrying information from one tag to another